whisper/whisperv5/filter.go

```diff
@@ -200,14 +202,27 @@ func (f *Filter) MatchTopic(topic TopicType) bool {
 		return true
 	}
 
-	for _, t := range f.Topics {
-		if t == topic {
+	for _, bt := range f.Topics {
+		if MatchSingleTopic(topic, bt) {
 			return true
 		}
 	}
 	return false
 }
 
+func MatchSingleTopic(topic TopicType, bt []byte) bool {
+	if len(bt) > 4 {
```

Comment:
topic in a message is 4 bytes (fixed). topic in filter was previously the same, but now i have introduced new feature -- variable topic size for filtering.